plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/panel/references/ReferencesResultsContainer.java

```diff
@@ -221,7 +241,7 @@ private void refreshReferenceKeyList() {
             }
         }
 
-        if (!allEntities.isEmpty()) {
+        if (allEntities != null && !allEntities.isEmpty()) {
```

Comment:
may be "!CommonUtils.isEmpty(allEntities)" as usual?

Reply:
Totally forgot about this method, thanks.